src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -32,7 +32,11 @@ private static IntPtr LoadNativeLibrary()
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
                 {
```

Comment:
Exclude dlopen & dlsym from the project file for netcoreapp3?

Reply:
Excluded with condition `!$(TargetGroup.StartsWith('netcoreapp3')) AND '$(TargetsUnix)' == 'true'` in `System.Drawing.Common.csproj` and `System.Data.Odbc.Tests.csproj`, but it seems like `System.Drawing.Common` is not configured to build specifically for `netcoreapp30` and it is building with `netcoreapp` target group.

Can the .NET Core 3.0 specific configuration be added to these two projects, so we can exercise the new code paths (guarded by `#if netcoreapp30` directive)?

Reply:
The `netcoreapp` configuration that's already there should be building for latest, ie: .NET Core 3.0. Maybe the ifdefs should be changed to just `netcoreapp`.